spec/unit/indirector/catalog/compiler_spec.rb

```diff
@@ -506,6 +506,7 @@ def expects_no_source_metadata
             .with_parameter(:checksum, checksum_type)
             .with_parameter(:checksum_value, sha)
             .with_parameter(:source, 'puppet:///modules/mymodule/config_file.txt')
+            .with_parameter(:content_uri, 'puppet:///modules/mymodule/config_file.txt')
```

Comment:
Nope, that's me mis-reading the ticket. I didn't realize content_uri was expected to be the expanded environment-relative path of the file (which makes sense now that I've realized it - static_file_content doesn't want to know where to find modules and munge paths)

I can adjust the inlining logic to do the right thing there - I guess looking at `metadata.full_path`, and grabbing the environment-relative portion of it as `content_uri` is the correct implementation?